aws-env.go

```diff
@@ -17,13 +23,19 @@ func main() {
 		return
 	}
 
-        recursivePtr := flag.Bool("recursive", false, "recursively process parameters on path")
-        flag.Parse()
+	recursivePtr := flag.Bool("recursive", false, "recursively process parameters on path")
+	convertCase := flag.String("case", upper, "Converts ENV Key to upper or lower case")
```

Comment:
"Case" should not be required and by default it should not change string-case as currently it breaks backward compatibility - it's now by default changing letters case to upper.